packages/datatrak-web/src/features/Tasks/TasksTable/AssignTaskModal.tsx

```diff
@@ -61,6 +133,34 @@ export const AssignTaskModal = ({ task, Button }: AssignTaskModalProps) => {
         isLoading={isLoading}
       >
         <Container>
+          <MetaDataContainer>
+            <Row>
+              <Column>
+                <ItemWrapper>
+                  <Title>Survey</Title>
+                  <Value>{task?.survey?.name}</Value>
+                </ItemWrapper>
+                <ItemWrapper>
+                  <Title>Repeating task</Title>
+                  <Value>{displayRepeatSchedule}</Value>
+                </ItemWrapper>
+              </Column>
+              <Column>
+                <ItemWrapper>
+                  <Title>Entity</Title>
+                  <Value>{task?.entity?.name}</Value>
+                </ItemWrapper>
+                <ItemWrapper>
+                  <Title>Due date</Title>
+                  <Value>{displayDate(task?.dueDate)}</Value>
+                </ItemWrapper>
+              </Column>
+            </Row>
+            <ItemWrapper>
+              <Title>Status</Title>
+              <StatusPill status={task?.taskStatus} />
```

Comment:
I could be missing something but I think that all of these optional chaining operators aren't necessary since the table row and modal will only be rendered after the task is fetched. That is also consistent with the TS interface above and so we might be able to remove all of these which would make this file a bit cleaner and clearer